walkthrough/index.html

```diff
@@ -8,7 +8,7 @@
 
 To make better use of the guide, here's a brief outline of how this guide is structured:
 
-- This guide is comprised of several tutorials. These outline the steps needed to get Exercism installed. Inside each tutorial are step-by-step instructions to get Exercism running, as well as steps to verify whether you've done the right thing.
+- This guide contains several tutorials. These outline the steps needed to get Exercism installed. Inside each tutorial are step-by-step instructions to get Exercism running, as well as steps to verify whether you've done the right thing.
```

Comment:
Apparently, the latest version of the walkthrough was never synced with this repo.